modules/install/pages/extensions.adoc

```diff
@@ -1,24 +1,316 @@
+
+* Look in the `JBOSS_HOME/standalone/configuration` directory. Notice there are four standalone configuration files:
+
+. **standalone.xml**: Compares to the default profile in domain.xml.
+
+. **standalone-ha.xml**: Compares to the ha profile in domain.xml.
+
+. **standalone-full.xml**: Compares to the full profile in domain.xml.
+
+. **standalone-full-ha.xml**: Compares to the full-ha profile in domain.xml.
+
+They are provided so a standalone server instance can easily be started with more or less subsystems available.
+
+[NOTE]
+====
+Be careful not to confuse a JBoss EAP profile with a Jakarta EE profile. A JBoss EAP profile is a collection of subsystem configurations that are used to define the capabilities and services of a JBoss EAP server instance. A Jakarta EE profile is a collection of Java EE standards.
+====
+
+* In a managed domain, the administrator can create new profiles, either from scratch, or cloned from the ones provided, and then customize the new profiles before associating them to their respective groups.
+
+* A profile can also be a child of another profile, thus inheriting the subsystem configuration from its parent profile, so common configurations can be shared and maintained in a single place.
+
+== Lab 1: Starting different Subsystems
+
+*Outcome*: In this lab, you start the standalone server instance using different subsystems available.
+
+* Navigate to the JBoss_Home directory.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ cd /opt/EAP-8.0.0/bin/
+[vagrant@server bin]$
+----
+
+* Start the JBoss EAP instance using the default `standalone.xml` to start most commonly used subsystems. To use a different configuration file, the `--server-config` or `-c` parameter can be used as follows:
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server bin]$ sudo -u jboss /bin/sh ./standalone.sh -c standalone.xml
+
+or
+
+[student@workstation ~]$ sudo -u jboss /bin/sh /opt/EAP-8.0.0/bin/standalone.sh  --server-config standalone.xml
+----
+
+Press `ctrl+C` to stop the JBoss instance.
+
+* Start the JBoss EAP instance using the `standalone-ha.xml` to start clustering capabilities.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server bin]$ sudo -u jboss /bin/sh ./standalone.sh -c standalone-ha.xml
+----
+
+Press `ctrl+C` to stop the JBoss instance.
+
+* Start the JBoss EAP instance using the `standalone-ha.xml` to start messaging-activemq capabilities.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server bin]$ sudo -u jboss /bin/sh ./standalone.sh -c standalone-full.xml
+----
+
+Press `ctrl+C` to stop the JBoss instance.
+
+* Start the JBoss EAP instance using the `standalone-ha.xml` to start messaging-activemq and clustering capabilities.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server bin]$ sudo -u jboss /bin/sh ./standalone.sh -c standalone-full-ha.xml
+----
+
+Press `ctrl+C` to stop the JBoss instance.
+
+
+== Lab 2: Configuring Subsystems
```

Comment:
keep this lab. remove lab 1. show how to start different profiles in lecture

Reply:
removed and modified